build.gradle

```diff
@@ -76,8 +71,37 @@ allprojects {
 
 
 jacoco {
-    toolVersion = "0.8.3"
+    toolVersion = "0.8.5"
+}
+
+jacocoTestReport {
+    reports {
+        xml.enabled false
+        csv.enabled false
+    }
+    afterEvaluate {
+        classDirectories.setFrom(files(classDirectories.files.collect {
+            fileTree(dir: it)
+        }))
+    }
+}
+test.finalizedBy(project.tasks.jacocoTestReport)
+jacocoTestCoverageVerification {
+    violationRules {
+        rule {
+            limit {
+                minimum = 0.5
```

Comment:
Might be worth adding a comment about test coverage in the ReadMe.

Reply:
Not sure if coverage needs to be added to README. Seems ok to me.

Reply:
Yes, probably not necessary.